tests/unit/test_main.py

```diff
@@ -205,57 +204,6 @@ async def test_handle_multipart_reply(
         await napp._handle_multipart_reply(ofpmp_desc, switch_one)
         assert switch_one.update_description.call_count == 1
 
-    @patch('napps.kytos.of_core.main.log')
-    @patch('kytos.core.buffers.KytosEventBuffer.aput')
-    @patch('napps.kytos.of_core.main.Main._update_switch_flows')
-    @patch('napps.kytos.of_core.v0x04.flow.Flow.from_of_flow_stats')
-    @patch('napps.kytos.of_core.main.Main._is_multipart_reply_ours')
-    async def test_on_multipart_flow_stats(
```

Comment:
@Alopalao answering the question you've asked on this PR description. This `test_on_multipart_flow_stats` we want to keep it, this is testing out `_handle_multipart_flow_stats` which is related to OF1.3 (0x04), so, could you revert the deletion of test case?

Reply:
Got it.